platform/darwin/src/NSCompoundPredicate+MGLAdditions.mm

```diff
@@ -0,0 +1,51 @@
+#import "NSCompoundPredicate+MGLAdditions.h"
+
+#import "NSPredicate+MGLAdditions.h"
+#import "NSExpression+MGLAdditions.h"
+
+class PredicateEvaluator {
+public:
+    
+    std::vector<mbgl::style::Filter>getFilters (NSArray<NSPredicate *> *predicates) {
```

Comment:
This function should become `-[NSCompoundPredicate(MGLAdditions) mgl_subfilters]`.